src/pages/study-room/provider/exit-room-modal-provier.tsx

```diff
@@ -0,0 +1,56 @@
+import Modal from '@/components/common/modal/modal';
+
+import { Typography } from '@mui/material';
+
+import { PropsWithChildren, createContext, useState } from 'react';
+
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { ROUTE_PATH } from '@/constant/routes';
+import useExitStudyRoom from '../hooks/useExitStudyRoom';
+
+const initialContext = {
+  isOpen: false,
+  openDialog: () => {},
+  closeDialog: () => {},
+};
+
+export const ExitRoomModalContext = createContext(initialContext);
+
+const ExitRoomModalProvier = ({ children }: PropsWithChildren) => {
```

Comment:
오타있네요~ VSCode의 `Code Spell Checker` 익스텐션을 사용하지 않으신다면 사용하시면 좋을 것 같습니다~
```suggestion
const ExitRoomModalProvider = ({ children }: PropsWithChildren) => {
```